### ISSUE TITLE
Plain text conformance statements/predicate "code"

### ISSUE THREAD

**Issue by carolinerosin** [haffonist/igamt#1938]
I can only add the description of the "plain text" constraint. It would be nice to be able to add the "code" as well. At least there should be a place holder in the exported XML file.

**Comment by Jungyubw** [comment]
@carolinerosin  
Not sure what you mean. Do you want to edit assertion code?

**Comment by carolinerosin** [comment]
yes, I mean being able to add the assertion code. If that is not possible, would be nice to have a "placeholder" in the constraint file (may require some changes in the schema and/or validation).

**Comment by ncrouzier-nist** [comment]
Bump!
